<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailWindow</class>
 <widget class="QWidget" name="DetailWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Detaylar</string>
  </property>
  <widget class="QTextBrowser" name="code1_simple">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>550</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser
{
	background: #FFF;
	padding: 10px;
}</string>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::WidgetWidth</enum>
   </property>
   <property name="lineWrapColumnOrWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QTextBrowser" name="code2_simple">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>240</y>
     <width>550</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser
{
	background: #FFF;
	padding: 10px;
}</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>20</width>
     <height>730</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Sadeleştirilmiş Kod:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>210</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Sadeleştirilmiş Kod:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Kod İskeleti:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="code1_skeleton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>550</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser
{
	background: #FFF;
	padding: 10px;
}</string>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::WidgetWidth</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>480</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Kod İskeleti:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="code2_skeleton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>510</y>
     <width>550</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser
{
	background: #FFF;
	padding: 10px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="code1_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	color: #00a8ff;
}</string>
   </property>
   <property name="text">
    <string>code1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="code2_label">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	color: #00a8ff;
}</string>
   </property>
   <property name="text">
    <string>code2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="simBar1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="value">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Benzerlik:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Benzerlik:</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="simBar2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="value">
    <number>50</number>
   </property>
  </widget>
  <widget class="QPushButton" name="code1_push">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>251</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background: #34495e;
	color: #FFF;
}</string>
   </property>
   <property name="text">
    <string>Kodu Görüntüleyin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="code2_push">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>130</y>
     <width>251</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background: #34495e;
	color: #FFF;
}</string>
   </property>
   <property name="text">
    <string>Kodu Görüntüleyin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Toplam parmak izi sayısı:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Seçilen parmak izi sayısı:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Eşleşen parmak izi sayısı:</string>
   </property>
  </widget>
  <widget class="QLabel" name="code1_totalFP">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: #FFF;
	border-radius: 1px solid black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="code1_selectedFP">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: #FFF;
	border-radius: 1px solid black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="code1_matchedFP">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: #FFF;
	border-radius: 1px solid black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>70</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Toplam parmak izi sayısı:</string>
   </property>
  </widget>
  <widget class="QLabel" name="code2_selectedFP">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>110</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: #FFF;
	border-radius: 1px solid black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>110</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Seçilen parmak izi sayısı:</string>
   </property>
  </widget>
  <widget class="QLabel" name="code2_matchedFP">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>150</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: #FFF;
	border-radius: 1px solid black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>150</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Eşleşen parmak izi sayısı:</string>
   </property>
  </widget>
  <widget class="QLabel" name="code2_totalFP">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>70</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background: #FFF;
	border-radius: 1px solid black;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
